.env.example

```
@@ -22,11 +22,27 @@ VITE_POSTHOG_HOST=https://app.posthog.com
+
 # Intercom
 # Replaces: VITE_APP_INTERCOM_APP_ID (deprecated — still works as fallback)
 VITE_INTERCOM_ENABLED=false
 VITE_INTERCOM_APP_ID=your-intercom-app-id-here
 
+# Pylon — https://docs.usepylon.com/pylon-docs/chat-widget/chat-setup
+# App ID comes from the Pylon dashboard under Chat Widget settings.
+# Leave "Identity Verification" OFF in the Pylon dashboard: the frontend does
+# not send a signed JWT yet (no backend signing endpoint exists).
+VITE_PYLON_ENABLED=false
+VITE_PYLON_APP_ID=your-pylon-app-id-here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_PYLON_APP_ID key should go before the VITE_PYLON_ENABLED key

(UnorderedKey)